Tests/SwiftDocCTests/Converter/DocumentationConverterTests.swift

```diff
@@ -12,6 +12,7 @@
 import XCTest
 @testable import SwiftDocC
 
+@available(*, deprecated, message: "This deprecated API will be removed after 6.2 is released")
```

Comment:
Interesting I've never see a unit test deprecated before. Does Xcode or SPM run these as usual? Is there an option to skip them? Do we need deprecation warnings during test runs?

Reply:
Ah. I usually copy and paste a template comment about this. I don't know why I didn't do that here.

Both Xcode and SwiftPM runs deprecated tests like this without skipping them and without raising deprecation warnings. The reason for marking a test as deprecated is that it allows the test to use other deprecated API without causing deprecation warnings.

I'll find that comment and add it to all the newly deprecated tests.

Reply:
I added a comment in https://github.com/swiftlang/swift-docc/pull/1059/commits/0d28c07f21a027c3d763fe0eec2c1f8cf5b2e87d to all the deprecated test (that didn't already have it) that explains that the deprecation only silenced other deprecation warnings and that it doesn't skip those tests.